src/ResourceManagement/RecoveryServices.Backup/RecoveryServices.Tests/app.config

```diff
@@ -42,7 +42,20 @@
     <add key="StorageAccountId" value="/subscriptions/f2edfd5d-5496-4683-b94f-b3588c579009/resourceGroups/shraccanrg/providers/Microsoft.Storage/storageAccounts/shraccanseastorv2" />
     <add key="SourceVmIdForRestore" value="/subscriptions/f2edfd5d-5496-4683-b94f-b3588c579009/resourceGroups/shraccanrg/providers/Microsoft.Compute/virtualMachines/shraccanv2" />
 
-  </appSettings>
+   <!-- AzureSql related -->
+    <add key="WorkloadTypeAzureSqlDb" value="AzureSqlDb" />
```

Comment:
This is a really bad idea - it makes it very difficult to run the tests.  The recommendation is  to generate these values inside the test setup